src/Action/RetrieveAutocompleteItemsAction.php

```diff
@@ -57,8 +58,13 @@ public function __invoke(Request $request): JsonResponse
             throw new AccessDeniedException();
         }
 
-        // subject will be empty to avoid unnecessary database requests and keep autocomplete function fast
-        $admin->setSubject($admin->getNewInstance());
+        try {
+            // subject will be empty to avoid unnecessary database requests and keep autocomplete function fast
+            $admin->setSubject($admin->getNewInstance());
+        } catch (AbstractClassException) {
+            // in case the subject is an abstract entity, we continue
+            // no-op
```

Comment:
```suggestion
            // in case if the subject is an abstract entity, we continue
```

Reply:
I agree that the `no-op` can be removed.

If a comment need to be added, it would be about "why it's still working without a subject set"

Reply:
Done